dotcom-rendering/src/components/Discussion/Comments.tsx

```diff
@@ -320,14 +315,12 @@ export const Comments = ({
 		onExpand();
 		setFilters(newFilterObject);
 	};
-
-	const onPageChange = (pageNumber: number) => {
+	useEffect(() => {
 		// Pagination also show when the view is not expanded so we want to expand when clicked
 		onExpand();
 		const element = document.getElementById('comment-filters');
 		element?.scrollIntoView();
-		setPage(pageNumber);
-	};
```

Comment:
I've moved these into a useeffect as these should happen as an effect of the page changing